### DISCUSSION TITLE
WiX 5.0.0: Marking assemblies for the GAC

### DISCUSSION THREAD

**Discussion by rowbot38** [wixtoolset/issues#8127] [Questions]
USing 3.14 and XSLT, I was able to mark assemblies during a harvest with Assembly=.net

How is this to be done with <Files  /> ?

We also have config files that are marked as Permanent and NoOverwrite also using XSLT.

**Comment by rowbot38** [comment]
Anyone know?

**Reply by bevanweiss** [reply]
Have you looked into the Files element?
https://docs.firegiant.com/wix/fivefour/

https://docs.firegiant.com/wix/schema/wxs/files/

**Reply by bevanweiss** [reply]
If Files doesn't do all you require, you could look in the FireGiant HeatWave Advanced Harvesting options
https://docs.firegiant.com/heatwave/build-tools/harvesting/comparisons/
